src/components/DateTime/DateTime.tsx

```diff
@@ -0,0 +1,175 @@
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import localizedFormat from "dayjs/plugin/localizedFormat";
+import relativeTime from "dayjs/plugin/relativeTime";
+import timezone from "dayjs/plugin/timezone";
+import updateLocale from "dayjs/plugin/updateLocale";
+import utc from "dayjs/plugin/utc";
+
+import { Container } from "@/components/Container/Container";
+import { Panel } from "@/components/Panel/Panel";
+import { Popover } from "@/components/Popover/Popover";
+import { Text } from "@/components/Typography/Text/Text";
+import styled from "styled-components";
+
+dayjs.extend(duration);
+dayjs.extend(localizedFormat);
+dayjs.extend(updateLocale);
+dayjs.extend(utc);
+
+const thresholds = [
+  { l: "s", r: 1, d: "second" },
+  { l: "ss", r: 56, d: "second" },
+  { l: "m", r: 90, d: "second" },
+  { l: "mm", r: 55, d: "minute" },
+  { l: "h", r: 90, d: "minute" },
+  { l: "hh", r: 22, d: "hour" },
+  { l: "d", r: 40, d: "hour" },
+  { l: "dd", r: 31, d: "day" },
+  { l: "M", r: 45, d: "day" },
+  { l: "MM", r: 11, d: "month" },
+  { l: "y", r: 17, d: "month" },
+  { l: "yy", r: 2, d: "year" },
+];
+
+dayjs.extend(relativeTime, { thresholds });
+
+dayjs.updateLocale("en", {
+  relativeTime: {
+    future: "In %s",
+    past: "%s ago",
+    s: "a few seconds",
+    ss: "%d seconds",
+    m: "1 minute",
+    mm: "%d minutes",
+    h: "1 hour",
+    hh: "%d hours",
+    d: "1 day",
+    dd: "%d days",
+    w: "1 week",
+    ww: "%d weeks",
+    M: "1 month",
+    MM: "%d months",
+    y: "1 year",
+    yy: "%d years",
+  },
+});
+
+const UnderlinedTrigger = styled(Popover.Trigger)`
+  text-decoration: wavy underline;
+`;
+
+const dateStyle = "medium";
+const timeStyle = "medium";
+
+const createBasicDateTimeFormatter = () => {
+  return new Intl.DateTimeFormat(undefined, {
+    dateStyle,
+    timeStyle,
+  });
+};
+
+const formatDateTime = (date: Date, locale?: Intl.Locale, timeZone?: string) => {
+  let dateTimeFormatter;
+  try {
+    dateTimeFormatter = new Intl.DateTimeFormat(locale, {
```

Comment:
@hoorayimhelping Need to make sure the Intl.DateTimeFormat matches the one we are using in unified console as we are using a third party package and may differ in format